### DISCUSSION TITLE
Group / sum transactions by category

### DISCUSSION THREAD

**Discussion by kolaente** [firefly-iii/firefly-iii#5545] [General]
Is there a way to query the sum of all transactions in a category over a set period of time?

I know I can see the expenses per category on the dashboard, but I'd like to see what the balance of a certain category is, including expenses and earnings.

**Comment by JC5** [comment]
Check out the Reports page.

**Reply by kolaente** [reply]
Awesome, exactly what I was looking for! Thanks!
